gitpod-io/website
# Link from Try Now, Sign Up, and Connect to repo buttons to gitpod.io/new

## Description
This changes the CTA buttons on the website to link to https://gitpod.io/new in the Dashboard where you can create a new Project.

- [x] Change "Try Now" button in hero
- [x] Change "Sign up" button(s) in the header
- [x] Change "Connect Git repository" git-provider buttons in [#get-started](https://www.gitpod.io/#get-started) section

All buttons should have analytics. 

**Preview:**  https://deploy-preview-1140--gitpod-kumquat.netlify.app/

## Related Issue(s)
Fixes #1139 which is part of https://github.com/gitpod-io/gitpod/issues/5071

## How to test
Clicking on Try Now or Sign Up or one of the Connect Git repository buttons, should open the dashboard
- logged in users should see the New Project page
- unauthenticated users should see the Login page

## Release Notes
```release-note
The "Try Now" and "Sign Up" and "Connect Git repository" buttons on the website homepage link to https://gitpod.io/new in the Dashboard where you can create a new Project.
```
<a href="https://gitpod.io/#https://github.com/gitpod-io/website/pull/1140"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>

<a href="https://gitpod-staging.com/#https://github.com/gitpod-io/website/pull/1140"><img src="https://gitpod-staging.com/button/open-in-gitpod.svg"/></a>
_(staging)_
